packages/admin/admin/src/dataGrid/usePersistentColumnState.tsx

```diff
@@ -43,7 +43,11 @@ const useVisibilityModelFromColumnMediaQueries = (columns: GridColDef[] | undefi
     return visibilityModel;
 };
 
-export function usePersistentColumnState(stateKey: string): Omit<DataGridProps, "rows" | "columns"> {
+type GridProps = Omit<DataGridProps, "rows" | "columns"> & {
+    apiRef: React.MutableRefObject<any>;
```

Comment:
I'm not sure how to define the type here without `any` as different grids (community/pro) have different `apiRef` types.

Reply:
Why not use `ReturnType<typeof useGridApiRef>` here as well?

Reply:
Unfortunately the type differs, depending on from which package `useGridApiRef` is imported (`@mui/x-data-grid`, `@mui/x-data-grid-pro`)

Using `any` the `apiRef` can be applied to all data-grids. When using `ReturnType<typeof useGridApiRef>` from `@mui/x-data-grid` it can only be applied to `DataGrid` but not `DataGridPro`.